docs/kratos/social-signin/05_generic.mdx

```diff
@@ -68,3 +68,79 @@ import JsonnetWarning from '../../_common/jsonnetwarning.mdx'
 
 <JsonnetWarning format="Jsonnet code snippets" use="data mapping" />
 ```
+
+## Configuration using the Ory CLI
+
+Follow these steps to add a generic provider as a social sign-in provider to your Ory Cloud project using the Ory CLI:
+
+1. Create a client at your generic provider to get a Client ID and Client Secret.
+2. Create a [Jsonnet code snippet](#data-mapping) to map the desired claims to the Ory Identity schema.
+3. Encode the Jsonnet snippet with [Base64](https://www.base64encode.org/) or host it under an URL accessible to Ory Cloud.
+4. Download the Identity Service config from your Ory Cloud project and save it to a file:
+
+   ```shell
+   ## List all available projects
```

Comment:
This comment is not a proper sentence, so it shouldn't be capitalized.

```suggestion
   ## list all available projects
```

Reply:
Please revert this to the way it's been before. One - consistency, two: 
https://learn.microsoft.com/en-us/style-guide/scannable-content/lists#capitalization